lib/analyzer.js

```diff
@@ -214,11 +222,10 @@ Analyzer.prototype._parseHTML = function _parseHTML(htmlImport,
   var depHrefs = [];
   var metadataLoaded = Promise.resolve(EMPTY_METADATA);
   var parsed;
-
   try {
-    parsed = importParse(htmlImport);
+    parsed = importParse(htmlImport, href);
   } catch (err) {
-    console.log(err);
+    console.log(err.stack);
```

Comment:
If someone doesn't .catch() their promise, the error is silent. This could generate a duplicate message, but it's helped me a lot in linter/vulcanize debugging.